--- mtools-3.9.8/configure.in.wiget	Mon Jul 16 14:51:26 2001
+++ mtools-3.9.8/configure.in	Mon Jul 16 14:52:48 2001
@@ -24,7 +24,6 @@
 
 dnl Check for configuration options
 dnl Enable OS/2 extended density format disks
-undefine([xdf])dnl
 AC_ARG_ENABLE(xdf,
 [  --enable-xdf           support for OS/2 extended density format disks],
 [if test x$enableval = xyes; then
@@ -34,7 +33,6 @@
 
 dnl Check for configuration options
 dnl Enable usage of vold on Solaris
-undefine([vold])dnl
 AC_ARG_ENABLE(vold,
 [  --enable-vold          compatibility with Solaris' vold],
 [if test x$enableval = xyes; then
@@ -44,7 +42,6 @@
 
 dnl Check for configuration options
 dnl Enable usage of vold on Solaris
-undefine([new-vold])dnl
 AC_ARG_ENABLE(new-vold,
 [  --enable-new-vold          compatibility with Solaris' vold, new version],
 [newVold=x$enableval
@@ -55,7 +52,6 @@
 
 dnl Check for configuration options
 dnl Debugging
-undefine([debug])dnl
 AC_ARG_ENABLE(debug,
 [  --enable-debug         debuging messages],
 [if test x$enableval = xyes; then
@@ -65,7 +61,6 @@
 
 dnl Check for configuration options
 dnl Raw terminal code (enabled by default)
-undefine([raw_term])dnl
 AC_ARG_ENABLE(raw_term,
 [  --enable-raw-term      raw terminal (readkey behaviour, default)],
 [if test x$enableval = xyes; then
