--- mtools-4.0.30/configure.in.orig	2021-06-19 07:13:34.056342392 +0200
+++ mtools-4.0.30/configure.in	2021-06-19 07:39:16.083031942 +0200
@@ -288,7 +288,7 @@
 CF_SYS_ERRLIST
 
 [
-host_os0=`echo $host_os | sed 's/-/_/g'`
+host_os0=`echo $host_os | sed 's/-gnu[a-z0-9]\+$/-gnu/; s/-/_/g'`
 host_os1=`echo $host_os0 | sed 's/\./_/g'`
 host_os2=`echo $host_os0 | sed 's/^\([^.]*\)\..*$/\1/g'`
 host_os3=`echo $host_os2 | sed 's/^\([^0-9]*\)[0-9]*$/\1/g'`
